packages/web-components/fast-components/docs/color.md

```diff
@@ -0,0 +1,143 @@
+# Color
+FAST utilizes an adaptive color system that provides some unique advantages:
+1. Ensure text meets [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) contrast requirements.
+2. Easily swap from light mode to dark, or anywhere in-between.
+3. Color themeing through palette tinting.
```

Comment:
I don't understand #3.  Could you rephrase please?  Bullets may be better representative of this type of content than numbers.

Reply:
@awentzel I'm not sure how to rephrase - to me that is pretty explanatory. Can you elaborate on what about it you don't understand? Maybe you have a suggestion?